<!DOCTYPE html>
<html lang="" xml:lang="">
<head>

  <meta charset="utf-8" />
  <meta http-equiv="X-UA-Compatible" content="IE=edge" />
  <title>5 Guerilla Analytics | COVID-19 Cases and Deaths in Netherlands in 2020</title>
  <meta name="description" content="5 Guerilla Analytics | COVID-19 Cases and Deaths in Netherlands in 2020" />
  <meta name="generator" content="bookdown 0.43 and GitBook 2.6.7" />

  <meta property="og:title" content="5 Guerilla Analytics | COVID-19 Cases and Deaths in Netherlands in 2020" />
  <meta property="og:type" content="book" />
  
  
  

  <meta name="twitter:card" content="summary" />
  <meta name="twitter:title" content="5 Guerilla Analytics | COVID-19 Cases and Deaths in Netherlands in 2020" />
  
  
  




  <meta name="viewport" content="width=device-width, initial-scale=1" />
  <meta name="apple-mobile-web-app-capable" content="yes" />
  <meta name="apple-mobile-web-app-status-bar-style" content="black" />
  
  
<link rel="prev" href="variant-analysis-tools.html"/>
<link rel="next" href="c.elegans-plate-experiment.html"/>
<script src="libs/jquery-3.6.0/jquery-3.6.0.min.js"></script>
<script src="https://cdn.jsdelivr.net/npm/fuse.js@6.4.6/dist/fuse.min.js"></script>
<link href="libs/gitbook-2.6.7/css/style.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-table.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-bookdown.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-highlight.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-search.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-fontsettings.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-clipboard.css" rel="stylesheet" />








<link href="libs/anchor-sections-1.1.0/anchor-sections.css" rel="stylesheet" />
<link href="libs/anchor-sections-1.1.0/anchor-sections-hash.css" rel="stylesheet" />
<script src="libs/anchor-sections-1.1.0/anchor-sections.js"></script>


<style type="text/css">
pre > code.sourceCode { white-space: pre; position: relative; }
pre > code.sourceCode > span { line-height: 1.25; }
pre > code.sourceCode > span:empty { height: 1.2em; }
.sourceCode { overflow: visible; }
code.sourceCode > span { color: inherit; text-decoration: inherit; }
pre.sourceCode { margin: 0; }
@media screen {
div.sourceCode { overflow: auto; }
}
@media print {
pre > code.sourceCode { white-space: pre-wrap; }
pre > code.sourceCode > span { display: inline-block; text-indent: -5em; padding-left: 5em; }
}
pre.numberSource code
  { counter-reset: source-line 0; }
pre.numberSource code > span
  { position: relative; left: -4em; counter-increment: source-line; }
pre.numberSource code > span > a:first-child::before
  { content: counter(source-line);
    position: relative; left: -1em; text-align: right; vertical-align: baseline;
    border: none; display: inline-block;
    -webkit-touch-callout: none; -webkit-user-select: none;
    -khtml-user-select: none; -moz-user-select: none;
    -ms-user-select: none; user-select: none;
    padding: 0 4px; width: 4em;
    color: #aaaaaa;
  }
pre.numberSource { margin-left: 3em; border-left: 1px solid #aaaaaa;  padding-left: 4px; }
div.sourceCode
  {   }
@media screen {
pre > code.sourceCode > span > a:first-child::before { text-decoration: underline; }
}
code span.al { color: #ff0000; font-weight: bold; } /* Alert */
code span.an { color: #60a0b0; font-weight: bold; font-style: italic; } /* Annotation */
code span.at { color: #7d9029; } /* Attribute */
code span.bn { color: #40a070; } /* BaseN */
code span.bu { color: #008000; } /* BuiltIn */
code span.cf { color: #007020; font-weight: bold; } /* ControlFlow */
code span.ch { color: #4070a0; } /* Char */
code span.cn { color: #880000; } /* Constant */
code span.co { color: #60a0b0; font-style: italic; } /* Comment */
code span.cv { color: #60a0b0; font-weight: bold; font-style: italic; } /* CommentVar */
code span.do { color: #ba2121; font-style: italic; } /* Documentation */
code span.dt { color: #902000; } /* DataType */
code span.dv { color: #40a070; } /* DecVal */
code span.er { color: #ff0000; font-weight: bold; } /* Error */
code span.ex { } /* Extension */
code span.fl { color: #40a070; } /* Float */
code span.fu { color: #06287e; } /* Function */
code span.im { color: #008000; font-weight: bold; } /* Import */
code span.in { color: #60a0b0; font-weight: bold; font-style: italic; } /* Information */
code span.kw { color: #007020; font-weight: bold; } /* Keyword */
code span.op { color: #666666; } /* Operator */
code span.ot { color: #007020; } /* Other */
code span.pp { color: #bc7a00; } /* Preprocessor */
code span.sc { color: #4070a0; } /* SpecialChar */
code span.ss { color: #bb6688; } /* SpecialString */
code span.st { color: #4070a0; } /* String */
code span.va { color: #19177c; } /* Variable */
code span.vs { color: #4070a0; } /* VerbatimString */
code span.wa { color: #60a0b0; font-weight: bold; font-style: italic; } /* Warning */
</style>

<style type="text/css">
  
  div.hanging-indent{margin-left: 1.5em; text-indent: -1.5em;}
</style>
<style type="text/css">
/* Used with Pandoc 2.11+ new --citeproc when CSL is used */
div.csl-bib-body { }
div.csl-entry {
  clear: both;
  margin-bottom: 0em;
}
.hanging div.csl-entry {
  margin-left:2em;
  text-indent:-2em;
}
div.csl-left-margin {
  min-width:2em;
  float:left;
}
div.csl-right-inline {
  margin-left:2em;
  padding-left:1em;
}
div.csl-indent {
  margin-left: 2em;
}
</style>

</head>

<body>



  <div class="book without-animation with-summary font-size-2 font-family-1" data-basepath=".">

    <div class="book-summary">
      <nav role="navigation">

<ul class="summary">
<li class="chapter" data-level="1" data-path="index.html"><a href="index.html"><i class="fa fa-check"></i><b>1</b> Introduction</a>
<ul>
<li class="chapter" data-level="1.1" data-path="index.html"><a href="index.html#contact"><i class="fa fa-check"></i><b>1.1</b> Contact</a></li>
</ul></li>
<li class="chapter" data-level="2" data-path="curriculum-vitae-wing-kien-alex-sung.html"><a href="curriculum-vitae-wing-kien-alex-sung.html"><i class="fa fa-check"></i><b>2</b> Curriculum Vitae Wing-kien Alex Sung</a>
<ul>
<li class="chapter" data-level="" data-path="curriculum-vitae-wing-kien-alex-sung.html"><a href="curriculum-vitae-wing-kien-alex-sung.html#education"><i class="fa fa-check"></i>Education</a></li>
<li class="chapter" data-level="" data-path="curriculum-vitae-wing-kien-alex-sung.html"><a href="curriculum-vitae-wing-kien-alex-sung.html#work-experience"><i class="fa fa-check"></i>Work Experience</a></li>
<li class="chapter" data-level="" data-path="curriculum-vitae-wing-kien-alex-sung.html"><a href="curriculum-vitae-wing-kien-alex-sung.html#activities"><i class="fa fa-check"></i>Activities</a></li>
<li class="chapter" data-level="" data-path="curriculum-vitae-wing-kien-alex-sung.html"><a href="curriculum-vitae-wing-kien-alex-sung.html#additional-skills"><i class="fa fa-check"></i>Additional Skills</a></li>
</ul></li>
<li class="chapter" data-level="3" data-path="looking-ahead.html"><a href="looking-ahead.html"><i class="fa fa-check"></i><b>3</b> Looking Ahead</a>
<ul>
<li class="chapter" data-level="" data-path="looking-ahead.html"><a href="looking-ahead.html#professional-development"><i class="fa fa-check"></i>Professional Development</a></li>
</ul></li>
<li class="chapter" data-level="4" data-path="variant-analysis-tools.html"><a href="variant-analysis-tools.html"><i class="fa fa-check"></i><b>4</b> Variant Analysis tools</a>
<ul>
<li class="chapter" data-level="4.1" data-path="variant-analysis-tools.html"><a href="variant-analysis-tools.html#summary-personal-timeline-and-motivation"><i class="fa fa-check"></i><b>4.1</b> Summary personal Timeline and Motivation</a></li>
<li class="chapter" data-level="4.2" data-path="variant-analysis-tools.html"><a href="variant-analysis-tools.html#background"><i class="fa fa-check"></i><b>4.2</b> Background</a></li>
<li class="chapter" data-level="4.3" data-path="variant-analysis-tools.html"><a href="variant-analysis-tools.html#discussion"><i class="fa fa-check"></i><b>4.3</b> Discussion</a>
<ul>
<li class="chapter" data-level="4.3.1" data-path="variant-analysis-tools.html"><a href="variant-analysis-tools.html#what-went-well"><i class="fa fa-check"></i><b>4.3.1</b> What Went Well</a></li>
<li class="chapter" data-level="4.3.2" data-path="variant-analysis-tools.html"><a href="variant-analysis-tools.html#what-didnt-work"><i class="fa fa-check"></i><b>4.3.2</b> What Didnâ€™t Work</a></li>
</ul></li>
<li class="chapter" data-level="4.4" data-path="variant-analysis-tools.html"><a href="variant-analysis-tools.html#conclusion-and-recommendations"><i class="fa fa-check"></i><b>4.4</b> Conclusion and Recommendations</a></li>
</ul></li>
<li class="chapter" data-level="5" data-path="guerilla-analytics.html"><a href="guerilla-analytics.html"><i class="fa fa-check"></i><b>5</b> Guerilla Analytics</a>
<ul>
<li class="chapter" data-level="5.1" data-path="guerilla-analytics.html"><a href="guerilla-analytics.html#the-focus-is-on-organizing-a-project-rnaseq-metagenomics-analysis"><i class="fa fa-check"></i><b>5.1</b> The focus is on organizing a project (RNAseq &amp; metagenomics analysis)</a></li>
<li class="chapter" data-level="5.2" data-path="guerilla-analytics.html"><a href="guerilla-analytics.html#organize-your-project"><i class="fa fa-check"></i><b>5.2</b> <strong>Organize Your Project</strong></a></li>
<li class="chapter" data-level="5.3" data-path="guerilla-analytics.html"><a href="guerilla-analytics.html#organisation-of-current-folder"><i class="fa fa-check"></i><b>5.3</b> Organisation of current folder</a></li>
</ul></li>
<li class="chapter" data-level="6" data-path="c.elegans-plate-experiment.html"><a href="c.elegans-plate-experiment.html"><i class="fa fa-check"></i><b>6</b> C.elegans Plate Experiment</a>
<ul>
<li class="chapter" data-level="6.1" data-path="c.elegans-plate-experiment.html"><a href="c.elegans-plate-experiment.html#exploring-the-excel-dataset-structure"><i class="fa fa-check"></i><b>6.1</b> Exploring the Excel Dataset Structure</a></li>
<li class="chapter" data-level="6.2" data-path="c.elegans-plate-experiment.html"><a href="c.elegans-plate-experiment.html#importing-the-dataset-into-r"><i class="fa fa-check"></i><b>6.2</b> Importing the Dataset into R</a></li>
<li class="chapter" data-level="6.3" data-path="c.elegans-plate-experiment.html"><a href="c.elegans-plate-experiment.html#checking-and-correcting-data-types"><i class="fa fa-check"></i><b>6.3</b> Checking and Correcting Data Types</a></li>
<li class="chapter" data-level="6.4" data-path="c.elegans-plate-experiment.html"><a href="c.elegans-plate-experiment.html#visualizing-raw-data-scatterplot"><i class="fa fa-check"></i><b>6.4</b> Visualizing Raw Data: Scatterplot</a></li>
<li class="chapter" data-level="6.5" data-path="c.elegans-plate-experiment.html"><a href="c.elegans-plate-experiment.html#fixing-axis-label-issues-in-ggplot"><i class="fa fa-check"></i><b>6.5</b> Fixing Axis Label Issues in ggplot</a></li>
<li class="chapter" data-level="6.6" data-path="c.elegans-plate-experiment.html"><a href="c.elegans-plate-experiment.html#identifying-positive-controls"><i class="fa fa-check"></i><b>6.6</b> Identifying Positive Controls</a></li>
<li class="chapter" data-level="6.7" data-path="c.elegans-plate-experiment.html"><a href="c.elegans-plate-experiment.html#identifying-negative-controls"><i class="fa fa-check"></i><b>6.7</b> Identifying Negative Controls</a></li>
<li class="chapter" data-level="6.8" data-path="c.elegans-plate-experiment.html"><a href="c.elegans-plate-experiment.html#designing-a-statistical-analysis-strategy"><i class="fa fa-check"></i><b>6.8</b> Designing a Statistical Analysis Strategy</a></li>
<li class="chapter" data-level="6.9" data-path="c.elegans-plate-experiment.html"><a href="c.elegans-plate-experiment.html#normalizing-data-using-the-control-group"><i class="fa fa-check"></i><b>6.9</b> Normalizing Data Using the Control Group</a></li>
<li class="chapter" data-level="6.10" data-path="c.elegans-plate-experiment.html"><a href="c.elegans-plate-experiment.html#why-normalize"><i class="fa fa-check"></i><b>6.10</b> Why Normalize?</a></li>
</ul></li>
<li class="chapter" data-level="7" data-path="reproducable-research-part-1.html"><a href="reproducable-research-part-1.html"><i class="fa fa-check"></i><b>7</b> Reproducable research part 1</a>
<ul>
<li class="chapter" data-level="7.1" data-path="reproducable-research-part-1.html"><a href="reproducable-research-part-1.html#heterologous-prime-boost-immunisation-with-mrna--and-adc68-based-2019-ncov-variant-vaccines-induces-broad-spectrum-immune-responses-in-mice"><i class="fa fa-check"></i><b>7.1</b> Heterologous prime-boost immunisation with mRNA- and AdC68-based 2019-nCoV variant vaccines induces broad-spectrum immune responses in mice</a>
<ul>
<li class="chapter" data-level="7.1.1" data-path="reproducable-research-part-1.html"><a href="reproducable-research-part-1.html#study-purpose"><i class="fa fa-check"></i><b>7.1.1</b> Study purpose</a></li>
<li class="chapter" data-level="7.1.2" data-path="reproducable-research-part-1.html"><a href="reproducable-research-part-1.html#study-location"><i class="fa fa-check"></i><b>7.1.2</b> Study location</a></li>
<li class="chapter" data-level="7.1.3" data-path="reproducable-research-part-1.html"><a href="reproducable-research-part-1.html#ethics-statement"><i class="fa fa-check"></i><b>7.1.3</b> Ethics statement</a></li>
<li class="chapter" data-level="7.1.4" data-path="reproducable-research-part-1.html"><a href="reproducable-research-part-1.html#funding-statement"><i class="fa fa-check"></i><b>7.1.4</b> Funding statement</a></li>
<li class="chapter" data-level="7.1.5" data-path="reproducable-research-part-1.html"><a href="reproducable-research-part-1.html#data-availability-statement-and-location"><i class="fa fa-check"></i><b>7.1.5</b> Data availability statement and location</a></li>
</ul></li>
<li class="chapter" data-level="7.2" data-path="reproducable-research-part-1.html"><a href="reproducable-research-part-1.html#estimating-the-effects-of-non-pharmaceutical-interventions-on-covid-19-in-europe"><i class="fa fa-check"></i><b>7.2</b> Estimating the effects of non-pharmaceutical interventions on COVID-19 in Europe</a></li>
<li class="chapter" data-level="7.3" data-path="reproducable-research-part-1.html"><a href="reproducable-research-part-1.html#estimated-impact-of-interventions"><i class="fa fa-check"></i><b>7.3</b> Estimated Impact of Interventions</a></li>
</ul></li>
<li class="chapter" data-level="8" data-path="relational-databases.html"><a href="relational-databases.html"><i class="fa fa-check"></i><b>8</b> Relational databases</a></li>
<li class="chapter" data-level="9" data-path="genetrackr.html"><a href="genetrackr.html"><i class="fa fa-check"></i><b>9</b> genetrackR</a></li>
<li class="chapter" data-level="10" data-path="optimising-contamination-control-strategies-in-the-cleanrooms-of-the-prinses-maxima-center.html"><a href="optimising-contamination-control-strategies-in-the-cleanrooms-of-the-prinses-maxima-center.html"><i class="fa fa-check"></i><b>10</b> Optimising contamination control strategies in the cleanrooms of the Prinses Maxima Center</a>
<ul>
<li class="chapter" data-level="10.1" data-path="optimising-contamination-control-strategies-in-the-cleanrooms-of-the-prinses-maxima-center.html"><a href="optimising-contamination-control-strategies-in-the-cleanrooms-of-the-prinses-maxima-center.html#covid-19-trend-in-netherlands-2020"><i class="fa fa-check"></i><b>10.1</b> COVID-19 Trend in Netherlands (2020)</a></li>
</ul></li>
<li class="chapter" data-level="11" data-path="references.html"><a href="references.html"><i class="fa fa-check"></i><b>11</b> References</a></li>
</ul>

      </nav>
    </div>

    <div class="book-body">
      <div class="body-inner">
        <div class="book-header" role="navigation">
          <h1>
            <i class="fa fa-circle-o-notch fa-spin"></i><a href="./">COVID-19 Cases and Deaths in Netherlands in 2020</a>
          </h1>
        </div>

        <div class="page-wrapper" tabindex="-1" role="main">
          <div class="page-inner">

            <section class="normal" id="section-">
<div id="guerilla-analytics" class="section level1 hasAnchor" number="5">
<h1><span class="header-section-number">5</span> Guerilla Analytics<a href="guerilla-analytics.html#guerilla-analytics" class="anchor-section" aria-label="Anchor link to header"></a></h1>
<p>The goal is to apply the <strong>Guerrilla Analytics</strong> framework to structure and document a project in a reproducible, collaborative, and future-proof way.</p>
<div id="the-focus-is-on-organizing-a-project-rnaseq-metagenomics-analysis" class="section level2 hasAnchor" number="5.1">
<h2><span class="header-section-number">5.1</span> The focus is on organizing a project (RNAseq &amp; metagenomics analysis)<a href="guerilla-analytics.html#the-focus-is-on-organizing-a-project-rnaseq-metagenomics-analysis" class="anchor-section" aria-label="Anchor link to header"></a></h2>
</div>
<div id="organize-your-project" class="section level2 hasAnchor" number="5.2">
<h2><span class="header-section-number">5.2</span> <strong>Organize Your Project</strong><a href="guerilla-analytics.html#organize-your-project" class="anchor-section" aria-label="Anchor link to header"></a></h2>
<p>I made a native folder and organised this with empty files in the way you would according to Guerilla analytics</p>
<p>To showcase your new project structure:</p>
<div class="sourceCode" id="cb6"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb6-1"><a href="guerilla-analytics.html#cb6-1" tabindex="-1"></a><span class="fu">library</span>(fs)</span></code></pre></div>
<pre><code>## Warning: package &#39;fs&#39; was built under R version 4.4.3</code></pre>
<div class="sourceCode" id="cb8"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb8-1"><a href="guerilla-analytics.html#cb8-1" tabindex="-1"></a><span class="co"># Print your current directory tree (recursively)</span></span>
<span id="cb8-2"><a href="guerilla-analytics.html#cb8-2" tabindex="-1"></a><span class="fu">dir_tree</span>(<span class="st">&quot;Opdracht 5 - Guerilla analytics&quot;</span>, <span class="at">recurse =</span> <span class="cn">TRUE</span>)</span></code></pre></div>
<pre><code>## Opdracht 5 - Guerilla analytics
## â”œâ”€â”€ 01_raw_data
## â”‚   â”œâ”€â”€ sample1_R1.fastq
## â”‚   â””â”€â”€ sample1_R2.fastq
## â”œâ”€â”€ 02_data
## â”‚   â””â”€â”€ sample1_aligned.bam
## â”œâ”€â”€ 03_analysis
## â”‚   â”œâ”€â”€ 01_import_and_clean.R
## â”‚   â”œâ”€â”€ 02_analysis.R
## â”‚   â””â”€â”€ 03_visualizations.R
## â”œâ”€â”€ 04_outputs
## â”‚   â”œâ”€â”€ figures
## â”‚   â”œâ”€â”€ reports
## â”‚   â””â”€â”€ tables
## â””â”€â”€ README.md</code></pre>
<hr />
</div>
<div id="organisation-of-current-folder" class="section level2 hasAnchor" number="5.3">
<h2><span class="header-section-number">5.3</span> Organisation of current folder<a href="guerilla-analytics.html#organisation-of-current-folder" class="anchor-section" aria-label="Anchor link to header"></a></h2>
<p>I also tried to make use of the guidelines of Guerilla analytics for this folder:</p>
<div class="sourceCode" id="cb10"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb10-1"><a href="guerilla-analytics.html#cb10-1" tabindex="-1"></a><span class="fu">library</span>(fs)</span>
<span id="cb10-2"><a href="guerilla-analytics.html#cb10-2" tabindex="-1"></a></span>
<span id="cb10-3"><a href="guerilla-analytics.html#cb10-3" tabindex="-1"></a><span class="co"># Print the directory tree of the current working directory</span></span>
<span id="cb10-4"><a href="guerilla-analytics.html#cb10-4" tabindex="-1"></a><span class="fu">dir_tree</span>(<span class="st">&quot;.&quot;</span>, <span class="at">recurse =</span> <span class="cn">TRUE</span>)</span></code></pre></div>
<pre><code>## .
## â”œâ”€â”€ 01_CV.Rmd
## â”œâ”€â”€ 02_Future.Rmd
## â”œâ”€â”€ 03_Variant-Calling.Rmd
## â”œâ”€â”€ 04_Guerilla-Analytics.Rmd
## â”œâ”€â”€ 05_CElegans.Rmd
## â”œâ”€â”€ 06_Reproducable-Research.Rmd
## â”œâ”€â”€ 07_Relational-data.Rmd
## â”œâ”€â”€ 08_genetrackR_for_RNA_seq_gene_expression_analysis.Rmd
## â”œâ”€â”€ 09_Optimising_Contamination_Control_Strategies.Rmd
## â”œâ”€â”€ 10_Parameterised_Covid_Data.Rmd
## â”œâ”€â”€ genetrackR
## â”‚   â”œâ”€â”€ countReads.R
## â”‚   â”œâ”€â”€ exportResults.R
## â”‚   â”œâ”€â”€ normalizeCounts.R
## â”‚   â”œâ”€â”€ plotGeneExpression.R
## â”‚   â””â”€â”€ Scripts
## â”œâ”€â”€ index.Rmd
## â”œâ”€â”€ LICENSE
## â”œâ”€â”€ Opdracht 1 - CV en Toekomstplan
## â”œâ”€â”€ Opdracht 2 - Vrije opdracht
## â”œâ”€â”€ Opdracht 3 Data Management
## â”‚   â”œâ”€â”€ Guerilla.R
## â”‚   â””â”€â”€ guerrilla-project
## â”‚       â”œâ”€â”€ 0_admin
## â”‚       â”‚   â””â”€â”€ README.md
## â”‚       â”œâ”€â”€ 1_data_raw
## â”‚       â”‚   â”œâ”€â”€ Data001
## â”‚       â”‚   â”‚   â”œâ”€â”€ 2024-01-01_cleanroom_counts.csv
## â”‚       â”‚   â”‚   â”œâ”€â”€ 2024-01-01_md5sum_cleanroom_counts.md5
## â”‚       â”‚   â”‚   â”œâ”€â”€ README.txt
## â”‚       â”‚   â”‚   â”œâ”€â”€ supporting
## â”‚       â”‚   â”‚   â”‚   â”œâ”€â”€ cleanroom_import.R
## â”‚       â”‚   â”‚   â”‚   â””â”€â”€ md5sum.R
## â”‚       â”‚   â”‚   â””â”€â”€ v01
## â”‚       â”‚   â”‚       â”œâ”€â”€ 2023-12-15_cleanroom_counts.csv
## â”‚       â”‚   â”‚       â””â”€â”€ 2023-12-15_md5sum_cleanroom_counts.md5
## â”‚       â”‚   â”œâ”€â”€ Data002
## â”‚       â”‚   â”‚   â”œâ”€â”€ experimental_design_notes.xlsx
## â”‚       â”‚   â”‚   â””â”€â”€ README.txt
## â”‚       â”‚   â””â”€â”€ README.txt
## â”‚       â”œâ”€â”€ 2_data_clean
## â”‚       â”‚   â””â”€â”€ cleaned_data_sample.csv
## â”‚       â”œâ”€â”€ 3_analysis
## â”‚       â”‚   â”œâ”€â”€ 01_import_and_clean.R
## â”‚       â”‚   â”œâ”€â”€ 02_analysis.R
## â”‚       â”‚   â””â”€â”€ 03_visualizations.R
## â”‚       â”œâ”€â”€ 5_meta
## â”‚       â”‚   â”œâ”€â”€ data_dictionary.md
## â”‚       â”‚   â””â”€â”€ variable_definitions.md
## â”‚       â”œâ”€â”€ 6_log
## â”‚       â”‚   â””â”€â”€ run_log.md
## â”‚       â””â”€â”€ guerrilla_project.Rproj
## â”œâ”€â”€ Opdracht 4 - Reproducible Science
## â”‚   â”œâ”€â”€ 4a - CELIQ
## â”‚   â”‚   â””â”€â”€ CE.LIQ.FLOW.062_Tidydata.xlsx
## â”‚   â””â”€â”€ 4b - Nature article files
## â”‚       â”œâ”€â”€ base-nature-2048690-mu.png
## â”‚       â”œâ”€â”€ base-nature.rds
## â”‚       â”œâ”€â”€ base-nature.stan
## â”‚       â”œâ”€â”€ covars-alpha-reductionbase-nature-2048690.png
## â”‚       â”œâ”€â”€ COVID-19-up-to-date.rds
## â”‚       â”œâ”€â”€ figures
## â”‚       â”‚   â”œâ”€â”€ Austria-three-pannel-base-nature-2048690.png
## â”‚       â”‚   â”œâ”€â”€ base-nature-2048690-final-rt.png
## â”‚       â”‚   â”œâ”€â”€ Belgium-three-pannel-base-nature-2048690.png
## â”‚       â”‚   â”œâ”€â”€ Denmark-three-pannel-base-nature-2048690.png
## â”‚       â”‚   â”œâ”€â”€ France-three-pannel-base-nature-2048690.png
## â”‚       â”‚   â”œâ”€â”€ Germany-three-pannel-base-nature-2048690.png
## â”‚       â”‚   â”œâ”€â”€ Italy-three-pannel-base-nature-2048690.png
## â”‚       â”‚   â”œâ”€â”€ Norway-three-pannel-base-nature-2048690.png
## â”‚       â”‚   â”œâ”€â”€ Spain-three-pannel-base-nature-2048690.png
## â”‚       â”‚   â”œâ”€â”€ Sweden-three-pannel-base-nature-2048690.png
## â”‚       â”‚   â”œâ”€â”€ Switzerland-three-pannel-base-nature-2048690.png
## â”‚       â”‚   â””â”€â”€ United Kingdom-three-pannel-base-nature-2048690.png
## â”‚       â”œâ”€â”€ interventions.rds
## â”‚       â”œâ”€â”€ popt-ifr.rds
## â”‚       â”œâ”€â”€ regions.rds
## â”‚       â”œâ”€â”€ Results
## â”‚       â”‚   â”œâ”€â”€ base-nature-2048690-stanfit.Rdata
## â”‚       â”‚   â”œâ”€â”€ cases-model-base-nature-2048690.csv
## â”‚       â”‚   â”œâ”€â”€ deaths-averted-base-nature-2048690.csv
## â”‚       â”‚   â”œâ”€â”€ deaths-model-base-nature-2048690.csv
## â”‚       â”‚   â”œâ”€â”€ deaths-null-base-nature-2048690.csv
## â”‚       â”‚   â””â”€â”€ per-cases-model-base-nature-2048690.csv
## â”‚       â”œâ”€â”€ scripts
## â”‚       â”‚   â”œâ”€â”€ covariate-size-effects.r
## â”‚       â”‚   â”œâ”€â”€ format-data.r
## â”‚       â”‚   â”œâ”€â”€ geom-stepribbon.r
## â”‚       â”‚   â”œâ”€â”€ make-table.r
## â”‚       â”‚   â”œâ”€â”€ plot-3-panel.r
## â”‚       â”‚   â””â”€â”€ process-covariates.r
## â”‚       â””â”€â”€ serial-interval.rds
## â”œâ”€â”€ Opdracht 5 - Guerilla analytics
## â”‚   â”œâ”€â”€ 01_raw_data
## â”‚   â”‚   â”œâ”€â”€ sample1_R1.fastq
## â”‚   â”‚   â””â”€â”€ sample1_R2.fastq
## â”‚   â”œâ”€â”€ 02_data
## â”‚   â”‚   â””â”€â”€ sample1_aligned.bam
## â”‚   â”œâ”€â”€ 03_analysis
## â”‚   â”‚   â”œâ”€â”€ 01_import_and_clean.R
## â”‚   â”‚   â”œâ”€â”€ 02_analysis.R
## â”‚   â”‚   â””â”€â”€ 03_visualizations.R
## â”‚   â”œâ”€â”€ 04_outputs
## â”‚   â”‚   â”œâ”€â”€ figures
## â”‚   â”‚   â”œâ”€â”€ reports
## â”‚   â”‚   â””â”€â”€ tables
## â”‚   â””â”€â”€ README.md
## â”œâ”€â”€ Portfolio-AlexS.Rproj
## â”œâ”€â”€ README.md
## â”œâ”€â”€ References.bib
## â”œâ”€â”€ _book
## â”‚   â”œâ”€â”€ 404.html
## â”‚   â”œâ”€â”€ bibliography.html
## â”‚   â”œâ”€â”€ c.elegans-plate-experiment.html
## â”‚   â”œâ”€â”€ creating-r-package.html
## â”‚   â”œâ”€â”€ curriculum-vitae-wing-kien-alex-sung.html
## â”‚   â”œâ”€â”€ genetrackr.html
## â”‚   â”œâ”€â”€ guerilla-analytics.html
## â”‚   â”œâ”€â”€ index.html
## â”‚   â”œâ”€â”€ libs
## â”‚   â”‚   â”œâ”€â”€ anchor-sections-1.1.0
## â”‚   â”‚   â”‚   â”œâ”€â”€ anchor-sections-hash.css
## â”‚   â”‚   â”‚   â”œâ”€â”€ anchor-sections.css
## â”‚   â”‚   â”‚   â””â”€â”€ anchor-sections.js
## â”‚   â”‚   â”œâ”€â”€ core-js-2.5.3
## â”‚   â”‚   â”‚   â””â”€â”€ shim.min.js
## â”‚   â”‚   â”œâ”€â”€ gitbook-2.6.7
## â”‚   â”‚   â”‚   â”œâ”€â”€ css
## â”‚   â”‚   â”‚   â”‚   â”œâ”€â”€ fontawesome
## â”‚   â”‚   â”‚   â”‚   â”‚   â””â”€â”€ fontawesome-webfont.ttf
## â”‚   â”‚   â”‚   â”‚   â”œâ”€â”€ plugin-bookdown.css
## â”‚   â”‚   â”‚   â”‚   â”œâ”€â”€ plugin-clipboard.css
## â”‚   â”‚   â”‚   â”‚   â”œâ”€â”€ plugin-fontsettings.css
## â”‚   â”‚   â”‚   â”‚   â”œâ”€â”€ plugin-highlight.css
## â”‚   â”‚   â”‚   â”‚   â”œâ”€â”€ plugin-search.css
## â”‚   â”‚   â”‚   â”‚   â”œâ”€â”€ plugin-table.css
## â”‚   â”‚   â”‚   â”‚   â””â”€â”€ style.css
## â”‚   â”‚   â”‚   â””â”€â”€ js
## â”‚   â”‚   â”‚       â”œâ”€â”€ app.min.js
## â”‚   â”‚   â”‚       â”œâ”€â”€ clipboard.min.js
## â”‚   â”‚   â”‚       â”œâ”€â”€ jquery.highlight.js
## â”‚   â”‚   â”‚       â”œâ”€â”€ plugin-bookdown.js
## â”‚   â”‚   â”‚       â”œâ”€â”€ plugin-clipboard.js
## â”‚   â”‚   â”‚       â”œâ”€â”€ plugin-fontsettings.js
## â”‚   â”‚   â”‚       â”œâ”€â”€ plugin-search.js
## â”‚   â”‚   â”‚       â””â”€â”€ plugin-sharing.js
## â”‚   â”‚   â”œâ”€â”€ htmltools-fill-0.5.8.1
## â”‚   â”‚   â”‚   â””â”€â”€ fill.css
## â”‚   â”‚   â”œâ”€â”€ htmlwidgets-1.6.4
## â”‚   â”‚   â”‚   â””â”€â”€ htmlwidgets.js
## â”‚   â”‚   â”œâ”€â”€ jquery-3.6.0
## â”‚   â”‚   â”‚   â””â”€â”€ jquery-3.6.0.min.js
## â”‚   â”‚   â”œâ”€â”€ react-18.2.0
## â”‚   â”‚   â”‚   â”œâ”€â”€ react-dom.min.js
## â”‚   â”‚   â”‚   â””â”€â”€ react.min.js
## â”‚   â”‚   â”œâ”€â”€ reactable-0.4.4
## â”‚   â”‚   â”‚   â””â”€â”€ reactable.css
## â”‚   â”‚   â”œâ”€â”€ reactable-binding-0.4.4
## â”‚   â”‚   â”‚   â””â”€â”€ reactable.js
## â”‚   â”‚   â””â”€â”€ reactwidget-2.0.0
## â”‚   â”‚       â””â”€â”€ react-tools.js
## â”‚   â”œâ”€â”€ looking-ahead.html
## â”‚   â”œâ”€â”€ Opdracht 4 - Reproducible Science
## â”‚   â”‚   â””â”€â”€ 4b - Nature article files
## â”‚   â”‚       â””â”€â”€ covars-alpha-reductionbase-nature-2048690.png
## â”‚   â”œâ”€â”€ optimisation.html
## â”‚   â”œâ”€â”€ optimising-contamination-control-strategies-in-the-cleanroom-of-the-prinses-maxima-center.html
## â”‚   â”œâ”€â”€ optimising-contamination-control-strategies-in-the-cleanrooms-of-the-prinses-maxima-center.html
## â”‚   â”œâ”€â”€ parameterized-covid.html
## â”‚   â”œâ”€â”€ references.html
## â”‚   â”œâ”€â”€ relational-databases.html
## â”‚   â”œâ”€â”€ reproducable-research-part-1.html
## â”‚   â”œâ”€â”€ resources.html
## â”‚   â”œâ”€â”€ search_index.json
## â”‚   â”œâ”€â”€ variant-analysis-tools.html
## â”‚   â””â”€â”€ _main_files
## â”‚       â””â”€â”€ figure-html
## â”‚           â”œâ”€â”€ ggplot-1.png
## â”‚           â”œâ”€â”€ normalised plot-1.png
## â”‚           â”œâ”€â”€ plot-cases-1.png
## â”‚           â”œâ”€â”€ plot-deaths-1.png
## â”‚           â””â”€â”€ Scatterplot-1.png
## â”œâ”€â”€ _bookdown_files
## â”œâ”€â”€ _main.Rmd
## â””â”€â”€ _main_files
##     â””â”€â”€ figure-html
##         â”œâ”€â”€ ggplot-1.png
##         â”œâ”€â”€ normalised plot-1.png
##         â”œâ”€â”€ plot-cases-1.png
##         â”œâ”€â”€ plot-deaths-1.png
##         â””â”€â”€ Scatterplot-1.png</code></pre>

</div>
</div>
            </section>

          </div>
        </div>
      </div>
<a href="variant-analysis-tools.html" class="navigation navigation-prev " aria-label="Previous page"><i class="fa fa-angle-left"></i></a>
<a href="c.elegans-plate-experiment.html" class="navigation navigation-next " aria-label="Next page"><i class="fa fa-angle-right"></i></a>
    </div>
  </div>
<script src="libs/gitbook-2.6.7/js/app.min.js"></script>
<script src="libs/gitbook-2.6.7/js/clipboard.min.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-search.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-sharing.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-fontsettings.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-bookdown.js"></script>
<script src="libs/gitbook-2.6.7/js/jquery.highlight.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-clipboard.js"></script>
<script>
gitbook.require(["gitbook"], function(gitbook) {
gitbook.start({
  "sharing": {
    "github": false,
    "facebook": true,
    "twitter": true,
    "linkedin": false,
    "weibo": false,
    "instapaper": false,
    "vk": false,
    "whatsapp": false,
    "all": ["facebook", "twitter", "linkedin", "weibo", "instapaper"]
  },
  "fontsettings": {
    "theme": "white",
    "family": "sans",
    "size": 2
  },
  "edit": {
    "link": null,
    "text": null
  },
  "history": {
    "link": null,
    "text": null
  },
  "view": {
    "link": null,
    "text": null
  },
  "download": null,
  "search": {
    "engine": "fuse",
    "options": null
  },
  "toc": {
    "collapse": "subsection"
  }
});
});
</script>

</body>

</html>
