<!DOCTYPE html>
<html lang="" xml:lang="">
<head>

  <meta charset="utf-8" />
  <meta http-equiv="X-UA-Compatible" content="IE=edge" />
  <title>10 Optimising Contamination Control Strategies in the cleanroom of the Prinses Maxima Center | My Projecticum Introductie</title>
  <meta name="description" content="10 Optimising Contamination Control Strategies in the cleanroom of the Prinses Maxima Center | My Projecticum Introductie" />
  <meta name="generator" content="bookdown 0.43 and GitBook 2.6.7" />

  <meta property="og:title" content="10 Optimising Contamination Control Strategies in the cleanroom of the Prinses Maxima Center | My Projecticum Introductie" />
  <meta property="og:type" content="book" />
  
  
  

  <meta name="twitter:card" content="summary" />
  <meta name="twitter:title" content="10 Optimising Contamination Control Strategies in the cleanroom of the Prinses Maxima Center | My Projecticum Introductie" />
  
  
  

<meta name="author" content="Alex Sung" />



  <meta name="viewport" content="width=device-width, initial-scale=1" />
  <meta name="apple-mobile-web-app-capable" content="yes" />
  <meta name="apple-mobile-web-app-status-bar-style" content="black" />
  
  
<link rel="prev" href="genetrackr.html"/>
<link rel="next" href="references.html"/>
<script src="libs/jquery-3.6.0/jquery-3.6.0.min.js"></script>
<script src="https://cdn.jsdelivr.net/npm/fuse.js@6.4.6/dist/fuse.min.js"></script>
<link href="libs/gitbook-2.6.7/css/style.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-table.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-bookdown.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-highlight.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-search.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-fontsettings.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-clipboard.css" rel="stylesheet" />








<link href="libs/anchor-sections-1.1.0/anchor-sections.css" rel="stylesheet" />
<link href="libs/anchor-sections-1.1.0/anchor-sections-hash.css" rel="stylesheet" />
<script src="libs/anchor-sections-1.1.0/anchor-sections.js"></script>



<style type="text/css">
  
  div.hanging-indent{margin-left: 1.5em; text-indent: -1.5em;}
</style>

</head>

<body>



  <div class="book without-animation with-summary font-size-2 font-family-1" data-basepath=".">

    <div class="book-summary">
      <nav role="navigation">

<ul class="summary">
<li class="chapter" data-level="1" data-path="index.html"><a href="index.html"><i class="fa fa-check"></i><b>1</b> Introduction</a>
<ul>
<li class="chapter" data-level="1.1" data-path="index.html"><a href="index.html#contact"><i class="fa fa-check"></i><b>1.1</b> Contact</a></li>
</ul></li>
<li class="chapter" data-level="2" data-path="curriculum-vitae-wing-kien-alex-sung.html"><a href="curriculum-vitae-wing-kien-alex-sung.html"><i class="fa fa-check"></i><b>2</b> Curriculum Vitae Wing-kien Alex Sung</a>
<ul>
<li class="chapter" data-level="" data-path="curriculum-vitae-wing-kien-alex-sung.html"><a href="curriculum-vitae-wing-kien-alex-sung.html#education"><i class="fa fa-check"></i>Education</a></li>
<li class="chapter" data-level="" data-path="curriculum-vitae-wing-kien-alex-sung.html"><a href="curriculum-vitae-wing-kien-alex-sung.html#work-experience"><i class="fa fa-check"></i>Work Experience</a></li>
<li class="chapter" data-level="" data-path="curriculum-vitae-wing-kien-alex-sung.html"><a href="curriculum-vitae-wing-kien-alex-sung.html#activities"><i class="fa fa-check"></i>Activities</a></li>
<li class="chapter" data-level="" data-path="curriculum-vitae-wing-kien-alex-sung.html"><a href="curriculum-vitae-wing-kien-alex-sung.html#additional-skills"><i class="fa fa-check"></i>Additional Skills</a></li>
</ul></li>
<li class="chapter" data-level="3" data-path="looking-ahead.html"><a href="looking-ahead.html"><i class="fa fa-check"></i><b>3</b> Looking Ahead</a>
<ul>
<li class="chapter" data-level="" data-path="looking-ahead.html"><a href="looking-ahead.html#professional-development"><i class="fa fa-check"></i>Professional Development</a></li>
</ul></li>
<li class="chapter" data-level="4" data-path="variant-analysis-tools.html"><a href="variant-analysis-tools.html"><i class="fa fa-check"></i><b>4</b> Variant Analysis tools</a>
<ul>
<li class="chapter" data-level="4.1" data-path="variant-analysis-tools.html"><a href="variant-analysis-tools.html#summary-personal-timeline-and-motivation"><i class="fa fa-check"></i><b>4.1</b> Summary personal Timeline and Motivation</a></li>
<li class="chapter" data-level="4.2" data-path="variant-analysis-tools.html"><a href="variant-analysis-tools.html#background"><i class="fa fa-check"></i><b>4.2</b> Background</a></li>
<li class="chapter" data-level="4.3" data-path="variant-analysis-tools.html"><a href="variant-analysis-tools.html#discussion"><i class="fa fa-check"></i><b>4.3</b> Discussion</a>
<ul>
<li class="chapter" data-level="4.3.1" data-path="variant-analysis-tools.html"><a href="variant-analysis-tools.html#what-went-well"><i class="fa fa-check"></i><b>4.3.1</b> What Went Well</a></li>
<li class="chapter" data-level="4.3.2" data-path="variant-analysis-tools.html"><a href="variant-analysis-tools.html#what-didnt-work"><i class="fa fa-check"></i><b>4.3.2</b> What Didn’t Work</a></li>
</ul></li>
<li class="chapter" data-level="4.4" data-path="variant-analysis-tools.html"><a href="variant-analysis-tools.html#conclusion-and-recommendations"><i class="fa fa-check"></i><b>4.4</b> Conclusion and Recommendations</a></li>
</ul></li>
<li class="chapter" data-level="5" data-path="guerilla-analytics.html"><a href="guerilla-analytics.html"><i class="fa fa-check"></i><b>5</b> Guerilla Analytics</a>
<ul>
<li class="chapter" data-level="5.0.1" data-path="guerilla-analytics.html"><a href="guerilla-analytics.html#reorganize-your-project"><i class="fa fa-check"></i><b>5.0.1</b> 1. <strong>Reorganize Your Project</strong></a></li>
<li class="chapter" data-level="5.0.2" data-path="guerilla-analytics.html"><a href="guerilla-analytics.html#apply-to-your-rnaseq-and-metagenomics-work"><i class="fa fa-check"></i><b>5.0.2</b> 2. <strong>Apply to Your RNAseq and Metagenomics Work</strong></a></li>
<li class="chapter" data-level="5.0.3" data-path="guerilla-analytics.html"><a href="guerilla-analytics.html#use-the-fs-package-to-generate-a-folder-tree-screenshot"><i class="fa fa-check"></i><b>5.0.3</b> 3. <strong>Use the <code>{fs}</code> Package to Generate a Folder Tree Screenshot</strong></a></li>
</ul></li>
<li class="chapter" data-level="6" data-path="c.elegans-plate-experiment.html"><a href="c.elegans-plate-experiment.html"><i class="fa fa-check"></i><b>6</b> C.elegans Plate Experiment</a>
<ul>
<li class="chapter" data-level="6.1" data-path="c.elegans-plate-experiment.html"><a href="c.elegans-plate-experiment.html#exploring-the-excel-dataset-structure"><i class="fa fa-check"></i><b>6.1</b> Exploring the Excel Dataset Structure</a></li>
<li class="chapter" data-level="6.2" data-path="c.elegans-plate-experiment.html"><a href="c.elegans-plate-experiment.html#importing-the-dataset-into-r"><i class="fa fa-check"></i><b>6.2</b> Importing the Dataset into R</a></li>
<li class="chapter" data-level="6.3" data-path="c.elegans-plate-experiment.html"><a href="c.elegans-plate-experiment.html#checking-and-correcting-data-types"><i class="fa fa-check"></i><b>6.3</b> Checking and Correcting Data Types</a></li>
<li class="chapter" data-level="6.4" data-path="c.elegans-plate-experiment.html"><a href="c.elegans-plate-experiment.html#visualizing-raw-data-scatterplot"><i class="fa fa-check"></i><b>6.4</b> Visualizing Raw Data: Scatterplot</a></li>
<li class="chapter" data-level="6.5" data-path="c.elegans-plate-experiment.html"><a href="c.elegans-plate-experiment.html#fixing-axis-label-issues-in-ggplot"><i class="fa fa-check"></i><b>6.5</b> Fixing Axis Label Issues in ggplot</a></li>
<li class="chapter" data-level="6.6" data-path="c.elegans-plate-experiment.html"><a href="c.elegans-plate-experiment.html#identifying-positive-controls"><i class="fa fa-check"></i><b>6.6</b> Identifying Positive Controls</a></li>
<li class="chapter" data-level="6.7" data-path="c.elegans-plate-experiment.html"><a href="c.elegans-plate-experiment.html#identifying-negative-controls"><i class="fa fa-check"></i><b>6.7</b> Identifying Negative Controls</a></li>
<li class="chapter" data-level="6.8" data-path="c.elegans-plate-experiment.html"><a href="c.elegans-plate-experiment.html#designing-a-statistical-analysis-strategy"><i class="fa fa-check"></i><b>6.8</b> Designing a Statistical Analysis Strategy</a></li>
<li class="chapter" data-level="6.9" data-path="c.elegans-plate-experiment.html"><a href="c.elegans-plate-experiment.html#normalizing-data-using-the-control-group"><i class="fa fa-check"></i><b>6.9</b> Normalizing Data Using the Control Group</a></li>
<li class="chapter" data-level="6.10" data-path="c.elegans-plate-experiment.html"><a href="c.elegans-plate-experiment.html#why-normalize"><i class="fa fa-check"></i><b>6.10</b> Why Normalize?</a></li>
</ul></li>
<li class="chapter" data-level="7" data-path="reproducable-research-part-1.html"><a href="reproducable-research-part-1.html"><i class="fa fa-check"></i><b>7</b> Reproducable research part 1</a>
<ul>
<li class="chapter" data-level="7.1" data-path="reproducable-research-part-1.html"><a href="reproducable-research-part-1.html#heterologous-prime-boost-immunisation-with-mrna--and-adc68-based-2019-ncov-variant-vaccines-induces-broad-spectrum-immune-responses-in-mice"><i class="fa fa-check"></i><b>7.1</b> Heterologous prime-boost immunisation with mRNA- and AdC68-based 2019-nCoV variant vaccines induces broad-spectrum immune responses in mice</a>
<ul>
<li class="chapter" data-level="7.1.1" data-path="reproducable-research-part-1.html"><a href="reproducable-research-part-1.html#study-purpose"><i class="fa fa-check"></i><b>7.1.1</b> Study purpose</a></li>
<li class="chapter" data-level="7.1.2" data-path="reproducable-research-part-1.html"><a href="reproducable-research-part-1.html#study-location"><i class="fa fa-check"></i><b>7.1.2</b> Study location</a></li>
<li class="chapter" data-level="7.1.3" data-path="reproducable-research-part-1.html"><a href="reproducable-research-part-1.html#ethics-statement"><i class="fa fa-check"></i><b>7.1.3</b> Ethics statement</a></li>
<li class="chapter" data-level="7.1.4" data-path="reproducable-research-part-1.html"><a href="reproducable-research-part-1.html#funding-statement"><i class="fa fa-check"></i><b>7.1.4</b> Funding statement</a></li>
<li class="chapter" data-level="7.1.5" data-path="reproducable-research-part-1.html"><a href="reproducable-research-part-1.html#data-availability-statement-and-location"><i class="fa fa-check"></i><b>7.1.5</b> Data availability statement and location</a></li>
</ul></li>
<li class="chapter" data-level="7.2" data-path="reproducable-research-part-1.html"><a href="reproducable-research-part-1.html#estimating-the-effects-of-non-pharmaceutical-interventions-on-covid-19-in-europe"><i class="fa fa-check"></i><b>7.2</b> Estimating the effects of non-pharmaceutical interventions on COVID-19 in Europe</a></li>
</ul></li>
<li class="chapter" data-level="8" data-path="relational-databases.html"><a href="relational-databases.html"><i class="fa fa-check"></i><b>8</b> Relational databases</a></li>
<li class="chapter" data-level="9" data-path="genetrackr.html"><a href="genetrackr.html"><i class="fa fa-check"></i><b>9</b> genetrackR</a></li>
<li class="chapter" data-level="10" data-path="optimising-contamination-control-strategies-in-the-cleanroom-of-the-prinses-maxima-center.html"><a href="optimising-contamination-control-strategies-in-the-cleanroom-of-the-prinses-maxima-center.html"><i class="fa fa-check"></i><b>10</b> Optimising Contamination Control Strategies in the cleanroom of the Prinses Maxima Center</a></li>
<li class="chapter" data-level="11" data-path="references.html"><a href="references.html"><i class="fa fa-check"></i><b>11</b> References</a></li>
</ul>

      </nav>
    </div>

    <div class="book-body">
      <div class="body-inner">
        <div class="book-header" role="navigation">
          <h1>
            <i class="fa fa-circle-o-notch fa-spin"></i><a href="./">My Projecticum Introductie</a>
          </h1>
        </div>

        <div class="page-wrapper" tabindex="-1" role="main">
          <div class="page-inner">

            <section class="normal" id="section-">
<div id="optimising-contamination-control-strategies-in-the-cleanroom-of-the-prinses-maxima-center" class="section level1 hasAnchor" number="10">
<h1><span class="header-section-number">10</span> Optimising Contamination Control Strategies in the cleanroom of the Prinses Maxima Center<a href="optimising-contamination-control-strategies-in-the-cleanroom-of-the-prinses-maxima-center.html#optimising-contamination-control-strategies-in-the-cleanroom-of-the-prinses-maxima-center" class="anchor-section" aria-label="Anchor link to header"></a></h1>
<p>Microbiological contamination remains one of the most critical risks in the manufacturing of Advanced Therapy Medicinal Products (ATMPs), where even minor microbial presence can compromise product quality and patient safety. ATMP production typically occurs under stringent Good Manufacturing Practice (GMP) conditions, and contamination control strategies must be robust, sensitive, and tailored to the facility’s microbial environment. Cleanroom Environmental Monitoring (EM) is a central component of microbiological control and is required by regulatory frameworks such as EU GMP Annex 1 to verify that classified areas maintain a state of control during manufacturing operations <span class="citation">[@sandleReviewCleanroomMicroflora2011]</span>.</p>
<p>Despite clear guidance, significant variation still exists in how EM protocols are implemented across pharmaceutical sites, particularly in incubation temperature, media type, and sample duration. For instance, while standard incubations at 30–35 °C are effective for human-associated bacteria, they often fail to recover molds that prefer lower temperatures <span class="citation">[@gordonComparisonDifferentIncubation2014]</span>. Dual-temperature incubation (e.g., 20–25 °C followed by 30–35 °C) is therefore often recommended to enhance detection, but this increases operational complexity and costs. In parallel, the use of media containing neutralizers is recommended to counteract disinfectant residues on sampled surfaces, which could otherwise inhibit microbial growth <span class="citation">[@morandiMicrobesIdentifiedMonitoring2024]</span>.</p>
<p>In addition to EM, another contamination risk in ATMP production is mycoplasma infection. Mycoplasmas are small, cell wall-deficient bacteria that can evade detection by traditional turbidity or microscopy-based techniques. Their contamination in cell culture systems can lead to altered cellular physiology, DNA damage, and invalidation of experimental results or therapeutic products. Regulatory authorities require mycoplasma testing for any product involving cell culture; however, conventional detection methods such as culture (28-day incubation) or PCR can be time-consuming or technically demanding. Rapid enzymatic screening methods like the MycoAlert™ assay provide a more user-friendly approach and can be effectively used in GMP workflows for routine screening <span class="citation">[@mollakazemihaSensitivityBiochemicalTest2014; @sandleMycoplasmaCellTherapy]</span>.</p>
<p>This project, conducted at the Prinses Máxima Center’s Cell and Gene Therapy Facility (CGTF), aimed to optimize contamination control strategies in newly operational GMP cleanrooms. Specifically, we tested and validated different EM media and incubation regimens, evaluated microbial profiles across multiple cleanroom grades and operational states, and assessed the performance of the MycoAlert™ enzymatic mycoplasma assay using both positive controls and uninfected cell lines. The methods were chosen to align with regulatory expectations while ensuring practicality and sensitivity. Our goal was to provide an evidence-based recommendation for routine microbiological monitoring in a high-stakes manufacturing environment.</p>
</div>
            </section>

          </div>
        </div>
      </div>
<a href="genetrackr.html" class="navigation navigation-prev " aria-label="Previous page"><i class="fa fa-angle-left"></i></a>
<a href="references.html" class="navigation navigation-next " aria-label="Next page"><i class="fa fa-angle-right"></i></a>
    </div>
  </div>
<script src="libs/gitbook-2.6.7/js/app.min.js"></script>
<script src="libs/gitbook-2.6.7/js/clipboard.min.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-search.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-sharing.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-fontsettings.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-bookdown.js"></script>
<script src="libs/gitbook-2.6.7/js/jquery.highlight.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-clipboard.js"></script>
<script>
gitbook.require(["gitbook"], function(gitbook) {
gitbook.start({
  "sharing": {
    "github": false,
    "facebook": true,
    "twitter": true,
    "linkedin": false,
    "weibo": false,
    "instapaper": false,
    "vk": false,
    "whatsapp": false,
    "all": ["facebook", "twitter", "linkedin", "weibo", "instapaper"]
  },
  "fontsettings": {
    "theme": "white",
    "family": "sans",
    "size": 2
  },
  "edit": {
    "link": null,
    "text": null
  },
  "history": {
    "link": null,
    "text": null
  },
  "view": {
    "link": null,
    "text": null
  },
  "download": null,
  "search": {
    "engine": "fuse",
    "options": null
  },
  "toc": {
    "collapse": "subsection"
  }
});
});
</script>

</body>

</html>
